nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-processor-configuration.js

```diff
@@ -214,6 +235,28 @@
             return true;
         }
 
+        // consider retried relationships
```

Comment:
There's an opportunity to keep things DRY here with the `marshalAutoTerminateRelationships` and `marshalRetriedRelationships` by creating one method that takes an argument to determine whether we are marshaling auto terminate or retry. Will update this when I get a chance.